kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/coordinator/group/GroupMetadata.java

```diff
@@ -166,11 +167,22 @@ public static class GroupSummary {
      */
     @Data
     static class CommitRecordMetadataAndOffset {
-        private final Optional<Long> appendedBatchOffset;
+        private final Optional<String> appendedBatchOffset;
```

Comment:
I'm more prefer to use `org.apache.bookkeeper.mledger.impl.PositionImpl` in here. 

Then we can use `this.appendedBatchOffset.get().compareTo(that.appendedBatchOffset.get()) < 0` to compare this two position.

Reply:
You are right